### PR TITLE
Swap fonts from `TT Firs Neue` to `DM Sans`

### LINKED ISSUE #2202: Hotswap app font to DM Sans
Similar to our 1:1 swap to TT First Neue during the reskin project, we have received the request to change our font to a more "generic" software font — something more immediately identifiable, less expressive, and more comfortable in a UI.

The design team settled on, and received approval for DM Sans, an open-source font from Google Fonts. Please update the font files in our theme with DM Sans, using the _variable_ version of the font for the most responsive compatibility.

ZIP File:
[DM_Sans.zip](https://github.com/user-attachments/files/16238382/DM_Sans.zip)

Google Fonts page:
https://fonts.google.com/specimen/DM+Sans/about?query=dm+sans

This should NOT touch Space Mono, only TT First Neue.

ALSO, engineering needs to properly set up DM Sans to be the default font for all text unless we explicitly say to use Space Mono.

### PR DESCRIPTION
**Pull request by mudrila**
Closes #2202

### CODE REVIEW

**Comment by tomstuart123** [comment]
note - won't have time to review this today. So could @decentdao/design act as non-technical review on this so its not blocked?

File: public/fonts/DMSans-VariableFont_opsz,wght.ttf
Comment (adamgall) [inline public/fonts/DMSans-VariableFont_opsz,wght.ttf:1]: dont need this file

**Review by adamgall — CHANGES_REQUESTED** [review]

**Review by xanaramoss — APPROVED** [review]

**Review by adamgall — APPROVED** [review]

File: public/fonts/DMSans-VariableFont_opsz,wght.ttf
Comment (Da-Colon) [reply public/fonts/DMSans-VariableFont_opsz,wght.ttf:1]: Isn't ttf font file. used to support older browsers?

Comment (adamgall) [reply public/fonts/DMSans-VariableFont_opsz,wght.ttf:1]: yeah but we haven't supported them in the past and i don't care about starting now.

**Pull request merged**